ADDING_TRANSLATIONS.md

```diff
@@ -0,0 +1,312 @@
++       namespaces: ["landing-page", "common", "tools", "newsletter"],
+        defaultNamespace: "landing-page",
+    },
+};
+```
+
+- Copy and add static site functions to the `newsletter/index.js` page.
+
+`pages` folder directory structure
+```diff
+  [lang]
++  ┣ newsletter
++  ┃ ┗ index.js
+   ┣ tools
+   ┃ ┗ cli.js
+   ┗ index.js
+```
+
+`newsletter/index.js`
+```diff
+...
++ import {
++   getAllLanguageSlugs,
++   getLanguage,
++   useTranslation
++ } from "../../lib/i18n";
+
+export default function NewsletterIndexPage() {
+
++  const { t } = useTranslation('landing-page');
+
+  return (
+    ...
+  );
+}
+
++ export async function getStaticPaths() {
++   const paths = getAllLanguageSlugs();
++   return {
++     paths,
++    fallback: false,
++  };
++ }
++ 
++ export async function getStaticProps({ params }) {
++   const language = getLanguage(params.lang);
++   return {
++     props: {
++      language,
++    },
++   };
++ }
+```
+
+- Add custom route `LinkComponent` wherever the `next/link` is used for routing to `/newsletter` href.
+
+`lib/i18nPaths.js`
+```diff
+const i18nPaths = {
+    en: [
+        "/tools/cli"
++       "/newsletter"
+    ],
+    de: [
+        "/tools/cli"
++       "/newsletter"
+    ]
+};
+
+export default i18nPaths;
+```
+
+You are now done with adding the localization to the `newsletter` page.
+
+**Make sure to fix the Cypress tests after using the `useTranslation()` hook inside any component that is being tested by Cypress.**
+
+## Adding a new locale:
+
+AsyncAPI welcomes people from all over the world irrespective of their languages.
+There exist a few locales like `en` (English) and `de` (German) which have available localizations present.
+
+If you want to introduce a new locale like `fr` so that the AsyncAPI website can then serve pages in the French locale, you would need to follow a series of steps.
+
+**Step 1: Create new JSON Files**
+- Navigate to the `locales` folder
+- Create a new folder with the name of the locale you want to introduce.
+- Create new `JSON` files with the same name as present in each of the other `locales` folders.
+- You may copy the existing `JSON` files present in the `en` folder. Change the values of those translation keys according to the new localization.
+
+**Step 2: Modify i18n configuration**
+- Navigate to the `next-i18next-static-site.config.js` file in the root of the project folder.
+- Add the name of the newly added `locale` to the `languages` array.
+
+**Step 3: Configure i18n routing**
+With the addition of a new internationalized page, you would need to make sure it is being served on the website when someone visits it.
+- Make sure to add the exact same `href` to the `lib/i18nPaths.js` in the respective locales which support that `href`.
+
+For example, if you have added `fr` locale and have completed translating the `tools/cli` page so that if someone clicks Tools -> CLI from the navigation menu, it redirects the user to the `asyncapi.com/fr/tools/cli` href.
```

Comment:
```suggestion
For example, if you have added `fr` locale and have completed translating the `tools/cli` page so that if someone clicks `Tools -> CLI` from the navigation menu, it redirects the user to the `asyncapi.com/fr/tools/cli` href.
```